README.md

```diff
@@ -20,27 +18,155 @@
+  // header by default (false means "don't send the header")
+  'Permissions-Policy': false,
+  // Don't send a "Referer" (sp) header unless the new URL shares the same
+  // origin. You can set this to `false` if you prefer cross-origin "Referer"
+  // headers be sent. Apostrophe does not rely on them
+  'Referrer-Policy': 'same-origin',
+  // `true` means it should be computed according to the rules below.
+  // You may also pass your own string, or `false` to not send this header.
+  // The `policies` option and all of its sub-options are ignored unless
+  // `Content-Security-Policy` is `true`.
+  'Content-Security-Policy': true,
+
+  // You do not have to copy and paste this entire example.
+  // The sub-options you specify for `policies` are intelligently merged
+  // with the defaults you see below. Any sub-option you specify explicitly at
+  // project level overrides all of the default settings shown below for that
+  // sub-option; you may set one to `false` to completely
+  // disable it. You may also introduce entirely new sub-options,
+  // which will also be honored.
+  //
+  // Policies of the same type from different sub-options are merged, with
+  // the largest set of keywords and hosts enabled. This is done because
+  // browsers do not support more than one style-src policy, for example, but
+  // do support specifying several hosts.
+  //
+  // Note the HOSTS wildcard which matches all expected hosts including CDN hosts
+  // and workflow hostnames.
+
+  policies: {
+    general: {
+      'default-src': `HOSTS`,
+      // Because it is necessary for some of the output of tiptap
+      'style-src': "HOSTS 'unsafe-inline'",
+      'script-src': `HOSTS`,
+      'font-src': `HOSTS`,
+      'frame-src': `'self'`
+    },
+
+    // Set this sub-option to false if you wish to forbid google fonts
+    googleFonts: {
+      'style-src': 'fonts.googleapis.com',
+      'font-src': 'fonts.gstatic.com'
+    },
+
+    // Set this sub-option to false if you do not use the video widget
+    oembed: {
+      'frame-src': '*.youtube.com *.vimeo.com'
+    },
+
+    // Set this sub-option to false if you do not wish to permit Google Analytics and
+    // Google Adsense
+    analytics: {
+      'default-src': '*.google-analytics.com *.doubleclick.net',
+      // Note that use of google tag manager by definition brings in scripts from
+      // more third party sites and you will need to add policies for them
+      'script-src': '*.google-analytics.com *.doubleclick.net *.googletagmanager.com',
+    }  
+  }
+```
+
+#### Inline style attributes are still allowed
+
+Note that `style-src` is set by default to permit inline style attributes. This is currently necessary
+because the output of the tiptap rich text editor used in Apostrophe involves inline
+styles in some cases.
+
+#### Inline script tags are **not** allowed
+
+Inline script tags are **not** allowed by default, as this is one of the primary benefits of using
```

Comment:
> are **not** allowed by default

By the headers? We can be clearer about that. You allude to CSP, but there's no reason to not be explicit about this stuff.